src/core/main_menu.cpp

```diff
@@ -36,6 +40,22 @@
 #endif
 
 
+/**********************************************************************
+**  Function: palnagothci_start
+**  Just run Palnagotchi
+**********************************************************************/
+void palnagotchi_start() {
+  #ifdef CARDPUTER
+    tft.fillScreen(BGCOLOR);
+    palnagotchi_setup();
+    delay(300); // Due to select button pressed to enter / quit this feature
+    while(!checkEscPress() && !checkSelPress()) {
+      palnagotchi_update();
+      delay(10);
+    }
+  #endif
+}
+
```

Comment:
Move this function to palnagotchi.h

and add the following lines before the `#endif` to release the allocated memory for sprites.
```
  canvas_top.deleteSprite();
  canvas_bot.deleteSprite();
  canvas_main.deleteSprite();
```

Reply:
Okay done